utils/api_info_v2.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Comment:
This is not a `v2` of the info api script, this is the first one. Plain `api_info.py` should be fine.